<ui version="4.0" >
 <class>MvdBrowserView</class>
 <widget class="QWidget" name="MvdBrowserView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QToolButton" name="backButton" >
     <property name="text" >
      <string>&lt;</string>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QToolButton" name="reloadButton" >
     <property name="text" >
      <string>R</string>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QToolButton" name="forwardButton" >
     <property name="text" >
      <string>></string>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>317</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QWebView" name="webView" >
     <property name="url" >
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
